indico/modules/events/timetable/controllers/manage.py

```diff
@@ -217,6 +226,35 @@ def _process_DELETE(self):
         delete_timetable_entry(self.break_.timetable_entry)
         return '', 204
 
+    @no_autoflush
+    def _change_parent(self, session_block_id):
+        entry = self.break_.timetable_entry
+        current_block = self.break_.session_block_obj
+
+        if session_block_id is None:
+            if current_block is None:
+                return
+            self.break_.session_block = None
+            self.break_.session = None
+            if entry.parent is not None:
+                update_timetable_entry(entry, {'parent': None, 'start_dt': entry.start_dt})
+        try:
+            target_id = int(session_block_id)
+        except (TypeError, ValueError):
+            raise BadRequest('Invalid session block id !!!!')
```

Comment:
`!!!!` 😂

Reply:
I don't think this check/conversion here is needed, because `session_block_id` can only be a number or `None`, but in case of `None` this should be handled before getting here